src/components/application_manager/src/mobile_command_factory.cc

```diff
@@ -133,6 +133,15 @@
 #include "application_manager/commands/mobile/send_location_response.h"
 #include "application_manager/commands/mobile/dial_number_request.h"
 #include "application_manager/commands/mobile/dial_number_response.h"
+#include "application_manager/commands/mobile/button_press_request.h"
```

Comment:
@LuxoftAKutsan, @okozlovlux, perhaps it should be included only if `SDL_REMOTE_CONTROL` defined